test/signs_ui_web/channels/user_socket_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule SignsUiWeb.UserSocketTest do
+  use SignsUiWeb.ChannelCase
+
+  setup do
+    %{socket: socket()}
+  end
+
+  describe "connect/2" do
+    test "authenticates when a valid token is given", %{socket: socket} do
+      current_time = DateTime.utc_now() |> DateTime.to_unix()
+      expiration_time = current_time + 100
```

Comment:
You might want to add more time to account for CI being slower.

Reply:
Bumped this up to 500 seconds in this and the other tests where we have a future expiration time.